homeassistant/components/zwave.py

```diff
@@ -175,10 +172,14 @@ def setup(hass, config):
     # pylint: disable=global-statement, import-error
     global NETWORK
 
+    import libopenzwave
```

Comment:
Since we can't install this using `REQUIREMENTS`, I would suggest we wrap this and print a proper warning to the user:
```python
try:
    import libopenzwave
except ImportError:
    _LOGGER.error("You are missing required dependency Python Open Z-Wave. Please follow instructions at X")
    return False
```